web/pages/cashout.tsx

```diff
@@ -137,6 +147,102 @@ const CashoutPage = () => {
     getCashoutSession(data)
   }
 
+  if (!user) {
+    return
+  }
+
+  const { status, message } = getVerificationStatus(user)
+
+  if (status !== 'success') {
+    return (
+      <Page trackPageView={'cashout page'}>
+        <Col className="mx-auto max-w-lg px-6 py-4">
+          {message == USER_NOT_REGISTERED_MESSAGE ||
+          message == PHONE_NOT_VERIFIED_MESSAGE ? (
+            <Col className="items-center gap-2">
+              <RegisterIcon height={40} className="fill-ink-700" />
+              <div className="text-2xl">You're not registered yet...</div>
+              <p className="text-ink-700 text-sm">
+                Registration is required to cash out.
+              </p>
+              <Link
+                href={'/gidx/register'}
+                className="bg-primary-500 hover:bg-primary-600 whitespace-nowrap rounded-lg px-4 py-2 text-white"
+              >
+                Register and get{' '}
+                <CoinNumber
+                  amount={KYC_VERIFICATION_BONUS}
+                  className={'font-bold'}
+                  isInline
+                />
+              </Link>
+            </Col>
+          ) : message == LOCATION_BLOCKED_MESSAGE ? (
```

Comment:
`LOCATION_BLOCKED_MESSAGE` is not returned anywhere. You need the private user to find out if their location is blocked, which would be a good thing to show! For that you have to use `locationBlocked`. While you're at it you could use `ageBlocked` and `identityBlocked` for even more informative messaging.